RPGManager_development/src/rpg_database/character_sheet/CharacterSheet.java

```diff
@@ -62,7 +66,7 @@ public String getEntryName() {
 		if (field.isContainted()) {
 			Object containerObject = this.characterData.get(field.getContainingField());
 			return (DataType) MultipleFieldsGetterSetter.class.cast(containerObject).getStoredValueByField(field);
-		} else {
+		}else {

```

Comment:
Ez csak egy apróság, de légyszi használd az eclipse autoformatterét :) Ha kell segítek beállítani. Egy csomó felesleges diffet ki lehet vele kerülni, ha ugyanazt a formattert használjuk.